app/dashboard/tests/test_dashboard_models.py

```diff
@@ -209,21 +209,6 @@ def test_can_submit_standard_bounty_after_expiration_date_if_deadline_extended()
         )
         assert bounty.can_submit_after_expiration_date is True
 
-    # @staticmethod
-    # def test_title_or_desc():
-    #     bounty = Bounty.objects.create(
-    #         title='TitleTest',
-    #         idx_status=0,
-    #         is_open=False,
-    #         web3_created=datetime(2008, 10, 31, tzinfo=pytz.UTC),
-    #         expires_date=datetime(2008, 11, 30, tzinfo=pytz.UTC),
-    #         github_url='https://github.com/gitcoinco/web/issues/1',
-    #         raw_data={}
-    #     )
-    #     assert bounty.title_or_desc == "TitleTest"
-    #     bounty.title = None
-    #     assert bounty.title_or_desc == "HTTP API Documentation"
-
```

Comment:
Was this failing? Why have we commented it out?

Reply:
He's removed out commented out code. :P